content/5_Plat/PIE.mdx

```diff
@@ -0,0 +1,360 @@
+---
+id: PIE
+title: 'Inclusion-Exclusion Principle'
+author: Mihnea Brebenel
+prerequisites:
+  - combo
+description:
+  'The inclusion-exclusion principle is a counting technique that generalizes the formula for computing the size of union of n finite sets.'
+frequency: 3
```

Comment:
this is supposed to reflect USACO frequency, but there aren't any example USACO problems in this module

Reply:
Ok i'll decrease the frequency